starter-code/scripts/article.js

```diff
@@ -19,10 +19,13 @@ Article.prototype.toHtml = function() {
   //   or say "(draft)" if it has no publication date:
   this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
   this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
-  // TODO: Use handlebars to render your articles!
+  // TODO: DONE! Use handlebars to render your articles!
   //       - Select your template from the DOM.
   //       - Now "compile" your template with Handlebars.
   //       - Don't forget to return your template for this article.
+
+  var template = Handlebars.compile($('#articleTemplate').html());
+  $('#articles').append(template(ourLocalData));
```

Comment:
If you were to use the #each helper method in Handlebars like you were attempting,  you could do it this way, but you should instead pass in the articles array with your actual articles instances rather than ourLocalData (and you would no longer need the forEach call at the bottom of the file).